src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -83,14 +89,51 @@ private liftHeights(double mPosition) {
         }
     }
 
+    public CommandBase disengageRatchet(DoubleSupplier speed) {
+        CommandBase cmd = new SequentialCommandGroup(moveTicks(.5, 0.10), turnOffBrake(), new WaitCommand(.1),
+                moveLift(speed));
+        cmd.setName("Disengage Ratchet");
+        return cmd;
+    }
+
+    public CommandBase turnOffBrake() {
+        return new InstantCommand(() -> elevatorBrake.set(true), this);
+    }
+
+    public CommandBase moveTicks(double ticks, double speed) {
+        CommandBase cmd = new FunctionalCommand(() -> {
+            liftEncoder.reset();
+        }, () -> {
+            SmartDashboard.putNumber("Lift Encoder", liftEncoder.getDistance());
+            elevatorMotor.set(speed);
+        }, (interrupted) -> {
+            elevatorMotor.stopMotor();
+            // This command might be interrupted if it takes to move the expected amount
+            // If that happens we should NOT continue the sequence as that could cause
+            // further damage.
+            if (interrupted) {
+                // This should get the command group that is trying to move the elevator
+                // We will then cancel that command group so we stop.
+                // TODO Send something to the dashboard.
+                CommandScheduler.getInstance().requiring(this).cancel();
```

Comment:
Did you test this?

Reply:
Nope

Reply:
It seems to work, though we didn't explicitly check it.

Reply:
Good enough for me.